### ISSUE TITLE
Website inaccessible because of certificate error

### ISSUE THREAD

**Issue by jolitp** [typst/webapp-issues#388]
### Description

From Firefox:

Websites prove their identity via certificates, which are valid for a set time period. The certificate for typst.app expired on 6/30/2024.

Error code: [SEC_ERROR_EXPIRED_CERTIFICATE](about:certerror?e=nssBadCert&u=https%3A//typst.app/docs/reference/styling/&s=badStsCert&c=UTF-8&d=%20#certificateErrorDebugInformation) 

From Chrome:

Your connection is not private
Attackers might be trying to steal your information from typst.app (for example, passwords, messages, or credit cards). [Learn more](chrome-error://chromewebdata/#)

NET::ERR_CERT_DATE_INVALID

### Reproduction URL

https://typst.app/

### Browsers

Firefox, Chrome

### OS

Linux

**Comment by zayndamji** [comment]
As described in [https://stackoverflow.com/a/47754532](https://stackoverflow.com/a/47754532), you can type "thisisunsafe" into the window and it will load the Typst.app webpages for you. While we wait for the Typst team to fix the issue, you can use this workaround.

**Comment by kwshi** [comment]
> As described in https://stackoverflow.com/a/47754532, you can type "thisisunsafe" into the window and it will load the Typst.app webpages for you. While we wait for the Typst team to fix the issue, you can use this workaround.

Per that StackOverflow post, this workaround is specifically for Chrome. Or at least I just tried it on Firefox and either I did it wrong or it doesn't work.

As a side note, the error I'm getting currently is not `SEC_ERROR_EXPIRED_CERTIFICATE` but rather `SEC_ERROR_UNKNOWN_ISSUER`.

![typst.app certificate](https://github.com/typst/webapp-issues/assets/3887189/67921b09-d819-4cec-874e-33e9841677a6)

![typst.app security error page in firefox](https://github.com/typst/webapp-issues/assets/3887189/6796ff34-6e4b-45a7-bcd1-d79bd73bcde0)

**Comment by zayndamji** [comment]
It seemed to work for me on Edge (chromium-based). Would you happen to have a Chromium-based browser that you could test it on?

**Comment by zjjott** [comment]
+1. typst.app certificate is out of date,
2024.7.1 07:59:59

**Comment by Microwave-WYB** [comment]
> It seemed to work for me on Edge (chromium-based). Would you happen to have a Chromium-based browser that you could test it on?

It's not really "work", it's more like Edge is allowing you to browse websites with an outdated certificate without warning 🤣.
Please renew the certificate. This may give users a bad impression of how you maintain your services.

**Comment by InvisOn** [comment]
My team just decided to move away from Typst because of this issue. :-(

**Comment by reknih** [comment]
We have renewed the certificate today at 7:50 CEST. Automatic renewal for Apex domain certificates is not possible using our current setup, we did not receive a notification from our operator Microsoft Azure prior to certificate expiry and were both asleep while it happened. We know this is not acceptable and have taken steps to mitigate -- I for one have set a manual calendar reminder for certificate renewal. Furthermore, we are setting up infrastructure such that our employees in other time zones can always call us when they become aware of an outage, including at night. I apologize for the inconvenience!
